crates/logger/Cargo.toml

```diff
@@ -9,3 +9,4 @@ console = "0.15.0"
 dirs = "4.0.0"
 fern = "0.6.1"
 log = "0.4.17"
+strip-ansi-escapes = "0.1.1"
```

Comment:
It looks like the `console` crate we're already using has a function for this: https://docs.rs/console/0.15.0/console/fn.strip_ansi_codes.html